sample-apps/endoscopy/README.md

```diff
@@ -131,6 +128,12 @@ monailabel start_server \
   --conf epistemic_top_k 3 \
   --conf auto_finetune_models tooltracking \
   --conf auto_finetune_check_interval 30
+
+# Active Learning for Inference
```

Comment:
Comment is something like this... (not really active learning for inference)
# Update/publish latest model back to CVAT

Reply:
Done